lib/suspenders/generators/jobs_generator.rb

```diff
@@ -3,7 +3,7 @@
 module Suspenders
   class JobsGenerator < Generators::Base
     def add_jobs_gem
-      append_file "Gemfile", %(\ngem "sidekiq"\n)
+      gem "sidekiq"
```

Comment:
Beautiful!

Reply:
That was my fault. I was accidentally on `main` when I pushed [that up](https://github.com/thoughtbot/suspenders/commit/e6e07b0db3ef6ffd0bfd1c3f70628dc8cd144c31) 🤦.

Is it worth removing that commit before I merge this in? 

Also, I wonder if we should enable a [branch protection rule](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule) to prevent this from happening again?

Reply:
Your choice!

Reply:
I think branch protection would be great! As for the commit, it's small and harmless, so I think we can leave it as-is. This PR makes a change to the same line, and also improves it, so that's good.